package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "test": "c8 mocha",
+    "testd": "c8 mocha --inspect-brk",
```

Comment:
do you really need this? usually this is automatically done via the IDE.

Reply:
sry for this , mistakely checked in